package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "d3": "^3.5.17",
     "lodash": "^4.13.1",
+    "mark.js": "github:julmot/mark.js#649fdcd",
```

Comment:
The GitHub syntax seems to not allow using a version tag here, so to make sure this works with future major releases a commit hash might be safer than master.